zerver/models.py

```diff
@@ -287,6 +288,14 @@ class Realm(models.Model):
         POLICY_NOBODY,
     ]
 
+    CREATE_WEB_PUBLIC_STREAM_POLICY_TYPES = [
+        POLICY_MEMBERS_ONLY,
+        POLICY_ADMINS_ONLY,
+        POLICY_FULL_MEMBERS_ONLY,
+        POLICY_MODERATORS_ONLY,
+        POLICY_OWNERS_ONLY,
+    ]
```

Comment:
@alya @amanagr @sahil839 did we have a decision on what we want the allowed options here to be?  One theory would be that "owners", "nobody", and "owners and admins" would be the only allowed options until we've figured out other product details.

(I think "nobody" is helpful for not having "web public stream" available in the UI at all when creating streams).

Reply:
Discussed this with Alya, and we'll allow only roles moderators and above for now.  I think that just involves changing this line and the API docs.

Reply:
Made this change, testing with coverage now.